### ISSUE TITLE
Linter stays half-dead on Windows (again?)

### ISSUE THREAD

**Issue by OverLordGoldDragon** [spyder-ide/spyder#13302]
[Original](https://github.com/spyder-ide/spyder/issues/11506); with the [patch](https://github.com/spyder-ide/spyder/pull/12771) applied, I get [this](https://puu.sh/G7nsU/2a829e2956.mp4):

```python
 File "D:\Anaconda\envs\tf2_env\lib\site-packages\spyder\plugins\completion\fallback\actor.py", line 155, in handle_msg
    msg['current_word'])
  File "D:\Anaconda\envs\tf2_env\lib\site-packages\spyder\plugins\completion\fallback\actor.py", line 62, in tokenize
    valid = is_prefix_valid(text, offset, language)
  File "D:\Anaconda\envs\tf2_env\lib\site-packages\spyder\plugins\completion\fallback\utils.py", line 125, in is_prefix_valid
    current_pos_text = text[offset - utf16_diff - 1]
IndexError: string index out of range
```

**Comment by OverLordGoldDragon** [comment]
@ccordoba12 This is the first time I've seen this error message - unsure if 4.14 will address it, but the patch didn't.

And nice bot.

**Comment by ccordoba12** [comment]
The error you reported here is unrelated to the linter staying dead, and it was fixed in 4.1.4. Please wait until it's released in Anaconda and try again.

**Comment by OverLordGoldDragon** [comment]
@ccordoba12 It has the same trigger (dot `__getattr__`), though admittedly I didn't check if it "stayed dead" (didn't seem to). We'll see in 4.14.
